roles/ceph-common/tasks/main.yml

```diff
@@ -0,0 +1,84 @@
+---
+- include: ./checks/check_system.yml
+
+- include: ./checks/check_mandatory_vars.yml
+
+- include: ./misc/system_tuning.yml
+  when: osd_group_name in group_names
+
+- include: ./pre_requisites/prerequisite_ice.yml
+  when: ceph_stable_ice
+
+- include: ./pre_requisites/prerequisite_rh_storage_iso_install.yml
+  when:
+    ceph_stable_rh_storage and
+    ceph_stable_rh_storage_iso_install
+
+- include: ./pre_requisites/prerequisite_rh_storage_cdn_install.yml
+  when:
+    ceph_stable_rh_storage and
+    ceph_stable_rh_storage_cdn_install
+
+- include: ./installs/install_on_redhat.yml
+  when: ansible_os_family == 'RedHat'
+
+- include: ./installs/install_on_debian.yml
+  when: ansible_os_family == 'Debian'
+
+- name: check for a ceph socket
+  shell: "stat /var/run/ceph/*.asok > /dev/null 2>&1"
+  changed_when: false
+  ignore_errors: true
+  register: socket
+
+- name: check for a rados gateway socket
+  shell: "stat {{ rbd_client_admin_socket_path }}*.asok > /dev/null 2>&1"
```

Comment:
There is a stat module that can be used here instead. Another upstream change to make if we're going forward with this role.